llvm/include/llvm/Frontend/OpenMP/OMP.td

```diff
@@ -730,15 +758,18 @@ def OMP_loop : Directive<"loop"> {
     VersionedClause<OMPC_Order, 50>,
   ];
   let association = AS_Loop;
+  let category = CA_Executable;
 }
 def OMP_masked : Directive<"masked"> {
   let allowedOnceClauses = [
     VersionedClause<OMPC_Filter>,
   ];
   let association = AS_Block;
+  let category = CA_Executable;
 }
 def OMP_Master : Directive<"master"> {
   let association = AS_Block;
+  let category = CA_Executable;
```

Comment:
This wasn't easy to find in a spec. The 5.2 spec has a category label in the table at the start of each section. But Master is deprecated in 5.2. Earlier specs don't show the category so clearly.

But executable sounds right to me

Reply:
I simply copied it from `masked`.